.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,70 @@
+<!--
+## Are you in the right place?
+1. For issues or feature requests, please create an issue in this repository.
+2. For general technical and non-technical questions, we are happy to help you on our [rook.io Slack](https://rook-io.slack.com).
+3. Did you already search the existing open issues for anything similar?
+-->
+
+<!-- If you have an issue, fill out the ISSUE. -->
+<!-- If you have a feature request delete the ISSUE part, uncomment and fill out the FEATURE REQUEST. -->
+<!-- ISSUE BEGIN -->
+## What did you do?
+<!--
+Exact steps of what you did, feel free to add code blocks of manifests here.
+-->
+1. ...
+2. ...
+3. ...
+
+### Expected behavior
+<!--
+What was the expected behavior by rook for you?
+-->
+
+### Behavior instead
+<!--
+What did rook instead of your expected behavior?
+Please let us know any circumstances for reproduction of this issue.
+-->
+
+<!--
+Having trouble with a certain component of rook?
+1. Use `kubectl logs -n ROOK_CLUSTER_NAMESPACE ROOK_POD_NAME`
+2. Copy the next lines and insert your logs in a code block.
+
+## Logs of COMPONENT_NAME:
+```
+__YOUR_LOGS_FOR_COMPONENT_HERE__
```

Comment:
I would like for users to always provide the Operator logs and agent logs since those are the main control paths for Rook.

Reply:
Added the section for the operator, so the user "always" provides them.